app/src/RoutesType/types.ts

```diff
@@ -0,0 +1,5 @@
+export type RootStackParamList = {
+  home: undefined;
+  facilities: undefined;
+  // Add other routes here
+};
```

Comment:
This is to defined types of routes to avoid ts errors. If route is not receiving any params then it is undefined, otherwise type of param should be defined